File: examples/anthropic-claude-code-in-sandbox-python/anthropic_claude_code_in_sandbox/build_prod.py
from dotenv import load_dotenv
from e2b import Template
from template import template, template_name

load_dotenv()

Template.build(
    template,
    alias=template_name,
    cpu_count=1,
    memory_mb=1024,
    on_build_logs=lambda log_entry: print(log_entry),
)

File: examples/anthropic-claude-code-in-sandbox-js/src/build.prod.ts
import "dotenv/config";
import { Template, defaultBuildLogger } from "e2b";
import { template, templateName } from "./template";

async function main() {
  await Template.build(template, {
    alias: templateName,
    cpuCount: 1,
    memoryMB: 1024,
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);
